Build failure: looking-glass-client

## Steps To Reproduce

Steps to reproduce the behavior:

1. `nix build github:nixos/nixpkgs/634fd46801442d760e09493a794c4f15db2d0cbb#looking-glass-client`

## Build log

<!-- insert build log in code block in collapsable section -->

<details>

<summary>Build Log</summary>

```
Running phase: unpackPhase
@nix { "action": "setPhase", "phase": "unpackPhase" }
unpacking source archive /nix/store/gxa54xkyx9482zac5q1nkjwgbjph1cq9-source
source root is source
Running phase: patchPhase
@nix { "action": "setPhase", "phase": "patchPhase" }
applying patch /nix/store/jcqa5fvn6v49b43nm0js26ca60c9gn0h-20972cfd9b940fddf9e7f3d2887a271d16398979.patch
patching file CMakeLists.txt
patching file displayservers/CMakeLists.txt
Running phase: updateAutotoolsGnuConfigScriptsPhase
@nix { "action": "setPhase", "phase": "updateAutotoolsGnuConfigScriptsPhase" }
Running phase: configurePhase
@nix { "action": "setPhase", "phase": "configurePhase" }
fixing cmake files...
cmake flags: -DCMAKE_FIND_USE_SYSTEM_PACKAGE_REGISTRY=OFF -DCMAKE_FIND_USE_PACKAGE_REGISTRY=OFF -DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON -DCMAKE_BUILD_TYPE=Release -DBUILD_TESTING=OFF -DCMAKE_INSTALL_LOCALEDIR=/nix/store/4cdb6mrr4cyif5k74wybnky2bxdc2gmx-looking-glass-client-B7-rc1/share/locale -DCMAKE_INSTALL_LIBEXECDIR=/nix/store/4cdb6mrr4cyif5k74wybnky2bxdc2gmx-looking-glass-client-B7-rc1/libexec -DCMAKE_INSTALL_LIBDIR=/nix/store/4cdb6mrr4cyif5k74wybnky2bxdc2gmx-looking-glass-client-B7-rc1/lib -DCMAKE_INSTALL_DOCDIR=/nix/store/4cdb6mrr4cyif5k74wybnky2bxdc2gmx-looking-glass-client-B7-rc1/share/doc/looking-glass-client -DCMAKE_INSTALL_INFODIR=/nix/store/4cdb6mrr4cyif5k74wybnky2bxdc2gmx-looking-glass-client-B7-rc1/share/info -DCMAKE_INSTALL_MANDIR=/nix/store/4cdb6mrr4cyif5k74wybnky2bxdc2gmx-looking-glass-client-B7-rc1/share/man -DCMAKE_INSTALL_OLDINCLUDEDIR=/nix/store/4cdb6mrr4cyif5k74wybnky2bxdc2gmx-looking-glass-client-B7-rc1/include -DCMAKE_INSTALL_INCLUDEDIR=/nix/store/4cdb6mrr4cyif5k74wybnky2bxdc2gmx-looking-glass-client-B7-rc1/include -DCMAKE_INSTALL_SBINDIR=/nix/store/4cdb6mrr4cyif5k74wybnky2bxdc2gmx-looking-glass-client-B7-rc1/sbin -DCMAKE_INSTALL_BINDIR=/nix/store/4cdb6mrr4cyif5k74wybnky2bxdc2gmx-looking-glass-client-B7-rc1/bin -DCMAKE_INSTALL_NAME_DIR=/nix/store/4cdb6mrr4cyif5k74wybnky2bxdc2gmx-looking-glass-client-B7-rc1/lib -DCMAKE_POLICY_DEFAULT_CMP0025=NEW -DCMAKE_FIND_FRAMEWORK=LAST -DCMAKE_STRIP=/nix/store/xcn9p4xxfbvlkpah7pwchpav4ab9d135-gcc-wrapper-14-20241116/bin/strip -DCMAKE_RANLIB=/nix/store/xcn9p4xxfbvlkpah7pwchpav4ab9d135-gcc-wrapper-14-20241116/bin/ranlib -DCMAKE_AR=/nix/store/xcn9p4xxfbvlkpah7pwchpav4ab9d135-gcc-wrapper-14-20241116/bin/ar -DCMAKE_C_COMPILER=gcc -DCMAKE_CXX_COMPILER=g++ -DCMAKE_INSTALL_PREFIX=/nix/store/4cdb6mrr4cyif5k74wybnky2bxdc2gmx-looking-glass-client-B7-rc1 -DOPTIMIZE_FOR_NATIVE=OFF
-- The C compiler identification is GNU 14.2.1
-- The CXX compiler identification is GNU 14.2.1
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /nix/store/xcn9p4xxfbvlkpah7pwchpav4ab9d135-gcc-wrapper-14-20241116/bin/gcc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /nix/store/xcn9p4xxfbvlkpah7pwchpav4ab9d135-gcc-wrapper-14-20241116/bin/g++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Performing Test COMPILER_SUPPORTS_MARCH_X86_64_V2
-- Performing Test COMPILER_SUPPORTS_MARCH_X86_64_V2 - Success
-- Found PkgConfig: /nix/store/0d4m43yp69lrm8imxbqgl9zxjwwz52jw-pkg-config-wrapper-0.29.2/bin/pkg-config (found version "0.29.2")
-- Checking for module 'fontconfig'
--   Found fontconfig, version 2.15.0
-- BFD libs: /nix/store/fw1wj32gjs0v8y23p86pma611vx63z2i-binutils-2.43.1-lib/lib/libbfd.so
-- Found BFD: /nix/store/fw1wj32gjs0v8y23p86pma611vx63z2i-binutils-2.43.1-lib/lib/libbfd.so
-- BFD libs: /nix/store/fw1wj32gjs0v8y23p86pma611vx63z2i-binutils-2.43.1-lib/lib/libbfd.so
-- Checking for modules 'spice-protocol;nettle;hogweed'
--   Found spice-protocol, version 0.14.4
--   Found nettle, version 3.10
--   Found hogweed, version 3.10
CMake Deprecation Warning at /build/source/repos/cimgui/CMakeLists.txt:1 (cmake_minimum_required):
  Compatibility with CMake < 3.5 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value or use a ...<max> suffix to tell
  CMake that the project does not need compatibility with older versions.


-- Checking for modules 'wayland-client;wayland-cursor;xkbcommon'
--   Found wayland-client, version 1.23.1
--   Found wayland-cursor, version 1.23.1
--   Found xkbcommon, version 1.7.0
-- Checking for modules 'wayland-client;wayland-cursor;xkbcommon'
--   Found wayland-client, version 1.23.1
--   Found wayland-cursor, version 1.23.1
--   Found xkbcommon, version 1.7.0
-- Checking for module 'libdecor-0'
--   No package 'libdecor-0' found
-- Checking for modules 'x11;xi;xfixes;xscrnsaver;xinerama;xcursor;xpresent;xkbcommon'
--   Found x11, version 1.8.10
--   Found xi, version 1.8.2
--   Found xfixes, version 6.0.1
--   Found xscrnsaver, version 1.2.4
--   Found xinerama, version 1.1.5
--   Found xcursor, version 1.2.3
--   Found xpresent, version 1.0.1
--   Found xkbcommon, version 1.7.0
-- Checking for modules 'egl;gl'
--   Found egl, version 1.5
--   Found gl, version 1.2
-- Checking for module 'wayland-egl'
--   Found wayland-egl, version 18.1.0
-- Using awk: /nix/store/a3c47r5z1q2c4rz0kvq8hlilkhx2s718-gawk-5.3.1/bin/gawk
-- Checking for module 'gl'
--   Found gl, version 1.2
-- Checking for module 'libpipewire-0.3'
--   Found libpipewire-0.3, version 1.2.7
-- Checking for module 'libpulse'
--   Found libpulse, version 17.0
-- Checking for module 'samplerate'
--   Found samplerate, version 0.2.2
-- The following features have been enabled:

 * ENABLE_OPENGL, Legacy OpenGL renderer.
 * ENABLE_EGL, EGL renderer.
 * ENABLE_BACKTRACE, Backtrace support.
 * ENABLE_PIPEWIRE, PipeWire audio support.
 * ENABLE_PULSEAUDIO, PulseAudio audio support.
 * ENABLE_X11, X11 support.
 * ENABLE_WAYLAND, Wayland support.

-- The following features have been disabled:

 * OPTIMIZE_FOR_NATIVE, Optimized for x86-64-v2
 * ENABLE_ASAN, AddressSanitizer support.
 * ENABLE_UBSAN, UndefinedBehaviorSanitizer support.
 * ENABLE_LIBDECOR, libdecor support.

-- Configuring done (1.5s)
-- Generating done (0.1s)
CMake Warning:
  Manually-specified variables were not used by the project:

    BUILD_TESTING
    CMAKE_EXPORT_NO_PACKAGE_REGISTRY
    CMAKE_POLICY_DEFAULT_CMP0025


-- Build files have been written to: /build/source/client/build
cmake: enabled parallel building
cmake: enabled parallel installing
Running phase: buildPhase
@nix { "action": "setPhase", "phase": "buildPhase" }
build flags: -j32 SHELL=/nix/store/5mh7kaj2fyv8mk4sfq1brwxgc02884wi-bash-5.2p37/bin/bash
[  0%] Creating object from lg-logo.svg
[  1%] Creating object from no-input-cursor/16.xcur
[  2%] Creating object from status/spice.svg
[  2%] Creating object from no-input-cursor/32.xcur
[  3%] Generating ../../../wayland/wayland-xdg-shell-client-protocol.h
[  4%] Generating ../../../wayland/wayland-idle-inhibit-unstable-v1-client-protocol.h
[  4%] Creating object from status/recording.svg
[  4%] Generating ../../../wayland/wayland-idle-inhibit-unstable-v1-client-protocol.c
[  5%] Generating ../../../wayland/wayland-keyboard-shortcuts-inhibit-unstable-v1-client-protocol.h
[  5%] Generating ../../../wayland/wayland-pointer-constraints-unstable-v1-client-protocol.c
[  5%] Generating ../../../wayland/wayland-keyboard-shortcuts-inhibit-unstable-v1-client-protocol.c
[  5%] Generating ../../../wayland/wayland-presentation-time-client-protocol.c
[  6%] Generating ../../../wayland/wayland-pointer-constraints-unstable-v1-client-protocol.h
[  7%] Generating ../../../wayland/wayland-presentation-time-client-protocol.h
[  7%] Generating ../../../wayland/wayland-relative-pointer-unstable-v1-client-protocol.c
[  8%] Generating ../../../wayland/wayland-relative-pointer-unstable-v1-client-protocol.h
[  9%] Generating ../../../wayland/wayland-viewporter-client-protocol.h
[  9%] Generating ../../../wayland/wayland-viewporter-client-protocol.c
[  9%] Generating ../../../wayland/wayland-xdg-activation-v1-client-protocol.c
[ 10%] Generating ../../../wayland/wayland-xdg-activation-v1-client-protocol.h
[ 10%] Generating ../../../wayland/wayland-xdg-decoration-unstable-v1-client-protocol.c
[ 11%] Generating ../../../wayland/wayland-xdg-decoration-unstable-v1-client-protocol.h
[ 11%] Generating ../../../wayland/wayland-xdg-output-unstable-v1-client-protocol.c
[ 12%] Generating ../../../wayland/wayland-xdg-output-unstable-v1-client-protocol.h
[ 12%] Generating ../../../wayland/wayland-xdg-shell-client-protocol.c
[ 12%] Building C object LGMP/CMakeFiles/lgmp.dir/src/host.c.o
[ 12%] Building C object LGMP/CMakeFiles/lgmp.dir/src/status.c.o
[ 13%] Building C object LGMP/CMakeFiles/lgmp.dir/src/client.c.o
[ 14%] Building C object PureSpice/CMakeFiles/purespice.dir/src/ps.c.o
[ 14%] Building C object common/src/platform/linux/CMakeFiles/lg_common_platform_code.dir/debug.c.o
[ 15%] Building CXX object cimgui/CMakeFiles/cimgui.dir/cimgui.cpp.o
[ 16%] Building C object common/src/platform/linux/CMakeFiles/lg_common_platform_code.dir/crash.c.o
[ 16%] Building C object common/src/platform/linux/CMakeFiles/lg_common_platform_code.dir/sysinfo.c.o
[ 16%] Building C object common/src/platform/linux/CMakeFiles/lg_common_platform_code.dir/thread.c.o
[ 17%] Building C object PureSpice/CMakeFiles/purespice.dir/src/rsa.c.o
[ 17%] Building C object PureSpice/CMakeFiles/purespice.dir/src/log.c.o
[ 17%] Building C object PureSpice/CMakeFiles/purespice.dir/src/queue.c.o
[ 18%] Building C object common/src/platform/linux/CMakeFiles/lg_common_platform_code.dir/event.c.o
[ 18%] Building CXX object cimgui/CMakeFiles/cimgui.dir/imgui/imgui.cpp.o
[ 19%] Building C object common/src/platform/linux/CMakeFiles/lg_common_platform_code.dir/ivshmem.c.o
[ 19%] Building CXX object cimgui/CMakeFiles/cimgui.dir/imgui/imgui_demo.cpp.o
[ 20%] Building C object common/src/platform/linux/CMakeFiles/lg_common_platform_code.dir/time.c.o
[ 21%] Building C object PureSpice/CMakeFiles/purespice.dir/src/channel.c.o
[ 21%] Building C object PureSpice/CMakeFiles/purespice.dir/src/channel_main.c.o
[ 22%] Building CXX object cimgui/CMakeFiles/cimgui.dir/imgui/imgui_tables.cpp.o
[ 22%] Building CXX object cimgui/CMakeFiles/cimgui.dir/imgui/imgui_draw.cpp.o
[ 22%] Building C object PureSpice/CMakeFiles/purespice.dir/src/channel_inputs.c.o
[ 22%] Building CXX object cimgui/CMakeFiles/cimgui.dir/imgui/imgui_widgets.cpp.o
[ 22%] Building C object common/src/platform/linux/CMakeFiles/lg_common_platform_code.dir/paths.c.o
[ 23%] Building C object common/src/platform/linux/CMakeFiles/lg_common_platform_code.dir/open.c.o
[ 23%] Building C object common/src/platform/linux/CMakeFiles/lg_common_platform_code.dir/cpuinfo.c.o
[ 24%] Building C object displayservers/Wayland/protocol/CMakeFiles/wayland_protocol.dir/__/__/__/wayland/wayland-xdg-shell-client-protocol.c.o
[ 24%] Building C object displayservers/Wayland/protocol/CMakeFiles/wayland_protocol.dir/__/__/__/wayland/wayland-presentation-time-client-protocol.c.o
[ 25%] Building C object displayservers/Wayland/protocol/CMakeFiles/wayland_protocol.dir/__/__/__/wayland/wayland-viewporter-client-protocol.c.o
[ 25%] Building C object displayservers/Wayland/protocol/CMakeFiles/wayland_protocol.dir/__/__/__/wayland/wayland-xdg-decoration-unstable-v1-client-protocol.c.o
[ 26%] Building C object PureSpice/CMakeFiles/purespice.dir/src/channel_playback.c.o
[ 26%] Building C object PureSpice/CMakeFiles/purespice.dir/src/channel_record.c.o
[ 27%] Building C object displayservers/Wayland/protocol/CMakeFiles/wayland_protocol.dir/__/__/__/wayland/wayland-relative-pointer-unstable-v1-client-protocol.c.o
[ 28%] Linking C static library liblg_resources.a
[ 28%] Building C object displayservers/Wayland/protocol/CMakeFiles/wayland_protocol.dir/__/__/__/wayland/wayland-pointer-constraints-unstable-v1-client-protocol.c.o
[ 29%] Building C object PureSpice/CMakeFiles/purespice.dir/src/channel_display.c.o
[ 30%] Building C object displayservers/Wayland/protocol/CMakeFiles/wayland_protocol.dir/__/__/__/wayland/wayland-keyboard-shortcuts-inhibit-unstable-v1-client-protocol.c.o
[ 30%] Built target lg_resources
[ 30%] Building C object PureSpice/CMakeFiles/purespice.dir/src/channel_cursor.c.o
[ 31%] Building C object PureSpice/CMakeFiles/purespice.dir/src/agent.c.o
[ 31%] Building C object displayservers/Wayland/protocol/CMakeFiles/wayland_protocol.dir/__/__/__/wayland/wayland-idle-inhibit-unstable-v1-client-protocol.c.o
[ 32%] Building C object displayservers/Wayland/protocol/CMakeFiles/wayland_protocol.dir/__/__/__/wayland/wayland-xdg-output-unstable-v1-client-protocol.c.o
[ 32%] Building C object displayservers/Wayland/protocol/CMakeFiles/wayland_protocol.dir/__/__/__/wayland/wayland-xdg-activation-v1-client-protocol.c.o
[ 33%] Linking C static library liblgmp.a
[ 34%] Linking C static library libwayland_protocol.a
[ 35%] Linking C static library liblg_common_platform_code.a
[ 35%] Built target lgmp
[ 35%] Built target wayland_protocol
[ 35%] Built target lg_common_platform_code
[ 36%] Building C object common/CMakeFiles/lg_common.dir/src/appstrings.c.o
[ 36%] Building C object common/CMakeFiles/lg_common.dir/src/stringutils.c.o
[ 39%] Building C object common/CMakeFiles/lg_common.dir/src/countedbuffer.c.o
[ 39%] Building C object common/CMakeFiles/lg_common.dir/src/framebuffer.c.o
[ 39%] Building C object common/CMakeFiles/lg_common.dir/src/stringlist.c.o
[ 40%] Building C object common/CMakeFiles/lg_common.dir/src/runningavg.c.o
[ 40%] Building C object common/CMakeFiles/lg_common.dir/src/rects.c.o
[ 40%] Building C object common/CMakeFiles/lg_common.dir/src/KVMFR.c.o
[ 40%] Building C object common/CMakeFiles/lg_common.dir/src/option.c.o
[ 40%] Building C object common/CMakeFiles/lg_common.dir/src/ringbuffer.c.o
[ 40%] Building C object common/CMakeFiles/lg_common.dir/src/cpuinfo.c.o
[ 41%] Building C object common/CMakeFiles/lg_common.dir/src/vector.c.o
[ 41%] Building C object common/CMakeFiles/lg_common.dir/src/ll.c.o
[ 42%] Building C object common/CMakeFiles/lg_common.dir/src/debug.c.o
[ 42%] Linking C static library libpurespice.a
[ 42%] Built target purespice
[ 43%] Linking C static library liblg_common.a
[ 43%] Built target lg_common
[ 43%] Building C object displayservers/X11/CMakeFiles/displayserver_X11.dir/x11.c.o
[ 44%] Building C object displayservers/X11/CMakeFiles/displayserver_X11.dir/atoms.c.o
[ 45%] Building C object displayservers/X11/CMakeFiles/displayserver_X11.dir/cursor.c.o
[ 45%] Building C object displayservers/X11/CMakeFiles/displayserver_X11.dir/wm/default.c.o
[ 45%] Building C object displayservers/X11/CMakeFiles/displayserver_X11.dir/clipboard.c.o
[ 46%] Building C object audiodevs/PulseAudio/CMakeFiles/audiodev_PulseAudio.dir/pulseaudio.c.o
[ 45%] Building C object audiodevs/PipeWire/CMakeFiles/audiodev_PipeWire.dir/pipewire.c.o
[ 47%] Building C object displayservers/Wayland/desktops/xdg/CMakeFiles/wayland_desktop_xdg.dir/xdg.c.o
[ 48%] Building C object displayservers/X11/CMakeFiles/displayserver_X11.dir/wm/i3.c.o
[ 48%] Linking C static library libaudiodev_PulseAudio.a
[ 48%] Linking C static library libwayland_desktop_xdg.a
[ 48%] Built target audiodev_PulseAudio
[ 48%] Built target wayland_desktop_xdg
[ 49%] Building C object displayservers/Wayland/desktops/CMakeFiles/wayland_desktops.dir/__/__/__/src/wayland_desktops.c.o
[ 49%] Linking C static library libwayland_desktops.a
[ 49%] Built target wayland_desktops
[ 49%] Building C object displayservers/Wayland/CMakeFiles/displayserver_Wayland.dir/cursor.c.o
[ 50%] Building C object displayservers/Wayland/CMakeFiles/displayserver_Wayland.dir/gl.c.o
[ 50%] Building C object displayservers/Wayland/CMakeFiles/displayserver_Wayland.dir/activation.c.o
[ 51%] Building C object displayservers/Wayland/CMakeFiles/displayserver_Wayland.dir/clipboard.c.o
[ 52%] Building C object displayservers/Wayland/CMakeFiles/displayserver_Wayland.dir/input.c.o
[ 53%] Building C object displayservers/Wayland/CMakeFiles/displayserver_Wayland.dir/poll.c.o
[ 53%] Building C object displayservers/Wayland/CMakeFiles/displayserver_Wayland.dir/output.c.o
[ 53%] Building C object displayservers/Wayland/CMakeFiles/displayserver_Wayland.dir/idle.c.o
[ 54%] Building C object displayservers/Wayland/CMakeFiles/displayserver_Wayland.dir/state.c.o
[ 54%] Building C object displayservers/Wayland/CMakeFiles/displayserver_Wayland.dir/presentation.c.o
[ 55%] Building C object displayservers/Wayland/CMakeFiles/displayserver_Wayland.dir/window.c.o
[ 55%] Building C object displayservers/Wayland/CMakeFiles/displayserver_Wayland.dir/wayland.c.o
[ 55%] Building C object displayservers/Wayland/CMakeFiles/displayserver_Wayland.dir/registry.c.o
[ 55%] Linking C static library libdisplayserver_X11.a
[ 56%] Linking C static library libdisplayserver_Wayland.a
[ 56%] Built target displayserver_X11
[ 56%] Built target displayserver_Wayland
[ 57%] Building C object displayservers/CMakeFiles/displayservers.dir/__/src/displayservers.c.o
[ 57%] Linking C static library libdisplayservers.a
[ 57%] Built target displayservers
[ 57%] Linking C static library libaudiodev_PipeWire.a
[ 57%] Built target audiodev_PipeWire
[ 58%] Building C object audiodevs/CMakeFiles/audiodevs.dir/__/src/audiodev.c.o
[ 58%] Linking C static library libaudiodevs.a
[ 58%] Built target audiodevs
[ 59%] Linking CXX static library cimgui.a
[ 59%] Built target cimgui
[ 59%] Preprocessing shader shader/basic.vert
[ 60%] Preprocessing shader shader/convert_24bit.frag
[ 60%] Preprocessing shader shader/ffx_fsr1_rcas.frag
[ 61%] Preprocessing shader shader/cursor.vert
[ 62%] Preprocessing shader shader/cursor_mono.frag
[ 62%] Preprocessing shader shader/cursor_rgb.frag
[ 63%] Preprocessing shader shader/damage.frag
[ 63%] Preprocessing shader shader/damage.vert
[ 64%] Preprocessing shader shader/desktop.vert
[ 64%] Preprocessing shader shader/desktop_rgb.frag
[ 64%] Preprocessing shader shader/downscale_lanczos2.frag
[ 65%] Preprocessing shader shader/downscale.frag
[ 66%] Preprocessing shader shader/downscale_linear.frag
[ 66%] Creating #defines from shader/desktop_rgb.frag
[ 66%] Preprocessing shader shader/ffx_cas.frag
[ 67%] Preprocessing shader shader/ffx_fsr1_easu.frag
[ 67%] Building C object renderers/OpenGL/CMakeFiles/renderer_OpenGL.dir/opengl.c.o
[ 68%] Building CXX object renderers/OpenGL/CMakeFiles/renderer_OpenGL.dir/build/source/repos/cimgui/imgui/backends/imgui_impl_opengl2.cpp.o
[ 68%] Creating object from shader/basic.vert
[ 69%] Creating object from shader/convert_24bit.frag
[ 70%] Creating object from shader/cursor.vert
[ 71%] Creating object from shader/cursor_mono.frag
[ 72%] Creating object from shader/damage.frag
[ 72%] Creating object from shader/cursor_rgb.frag
[ 73%] Creating object from shader/desktop.vert
[ 73%] Creating object from shader/damage.vert
[ 74%] Creating object from shader/downscale.frag
[ 74%] Creating object from shader/desktop_rgb.frag
[ 74%] Creating object from shader/downscale_lanczos2.frag
[ 75%] Creating object from shader/downscale_linear.frag
[ 75%] Creating object from shader/ffx_fsr1_rcas.frag
[ 76%] Creating object from shader/ffx_fsr1_easu.frag
[ 76%] Creating object from shader/ffx_cas.frag
[ 77%] Building C object renderers/EGL/CMakeFiles/renderer_EGL.dir/egldebug.c.o
[ 77%] Building C object renderers/EGL/CMakeFiles/renderer_EGL.dir/egl.c.o
[ 78%] Building C object renderers/EGL/CMakeFiles/renderer_EGL.dir/texture_util.c.o
[ 78%] Building C object renderers/EGL/CMakeFiles/renderer_EGL.dir/shader.c.o
[ 78%] Building C object renderers/EGL/CMakeFiles/renderer_EGL.dir/texture.c.o
[ 79%] Building C object renderers/EGL/CMakeFiles/renderer_EGL.dir/texture_buffer.c.o
[ 80%] Building C object renderers/EGL/CMakeFiles/renderer_EGL.dir/texture_dmabuf.c.o
[ 80%] Building C object renderers/EGL/CMakeFiles/renderer_EGL.dir/texture_framebuffer.c.o
[ 80%] Building C object renderers/EGL/CMakeFiles/renderer_EGL.dir/desktop_rects.c.o
[ 80%] Building C object renderers/EGL/CMakeFiles/renderer_EGL.dir/model.c.o
[ 80%] Building C object renderers/EGL/CMakeFiles/renderer_EGL.dir/damage.c.o
[ 81%] Building C object renderers/EGL/CMakeFiles/renderer_EGL.dir/desktop.c.o
[ 82%] Building C object renderers/EGL/CMakeFiles/renderer_EGL.dir/cursor.c.o
[ 82%] Building C object renderers/EGL/CMakeFiles/renderer_EGL.dir/framebuffer.c.o
[ 83%] Building C object renderers/EGL/CMakeFiles/renderer_EGL.dir/postprocess.c.o
[ 83%] Building C object renderers/EGL/CMakeFiles/renderer_EGL.dir/filter_24bit.c.o
[ 83%] Building C object renderers/EGL/CMakeFiles/renderer_EGL.dir/ffx.c.o
[ 83%] Building C object renderers/EGL/CMakeFiles/renderer_EGL.dir/filter_ffx_fsr1.c.o
[ 84%] Building C object renderers/EGL/CMakeFiles/renderer_EGL.dir/filter.c.o
[ 85%] Building C object renderers/EGL/CMakeFiles/renderer_EGL.dir/filter_ffx_cas.c.o
[ 86%] Building C object renderers/EGL/CMakeFiles/renderer_EGL.dir/filter_downscale.c.o
[ 86%] Building CXX object renderers/EGL/CMakeFiles/renderer_EGL.dir/build/source/repos/cimgui/imgui/backends/imgui_impl_opengl3.cpp.o
[ 86%] Linking CXX static library librenderer_OpenGL.a
[ 86%] Built target renderer_OpenGL
[ 87%] Linking CXX static library librenderer_EGL.a
[ 87%] Built target renderer_EGL
[ 88%] Building C object renderers/CMakeFiles/renderers.dir/__/src/renderers.c.o
[ 88%] Linking CXX static library librenderers.a
[ 88%] Built target renderers
[ 88%] Generating version.c, _version.c
[ 88%] Generating version.c, _version.c
[ 89%] Building C object CMakeFiles/looking-glass-client.dir/src/core.c.o
[ 89%] Building C object CMakeFiles/looking-glass-client.dir/src/main.c.o
[ 89%] Building C object CMakeFiles/looking-glass-client.dir/src/app.c.o
[ 89%] Building C object CMakeFiles/looking-glass-client.dir/src/config.c.o
[ 89%] Building C object CMakeFiles/looking-glass-client.dir/src/util.c.o
[ 90%] Building C object CMakeFiles/looking-glass-client.dir/src/keybind.c.o
[ 91%] Building C object CMakeFiles/looking-glass-client.dir/src/audio.c.o
[ 92%] Building C object CMakeFiles/looking-glass-client.dir/src/eglutil.c.o
[ 92%] Building C object CMakeFiles/looking-glass-client.dir/src/kb.c.o
[ 93%] Building C object CMakeFiles/looking-glass-client.dir/src/clipboard.c.o
[ 93%] Building C object CMakeFiles/looking-glass-client.dir/src/egl_dynprocs.c.o
[ 94%] Building C object CMakeFiles/looking-glass-client.dir/src/gl_dynprocs.c.o
[ 95%] Building C object CMakeFiles/looking-glass-client.dir/src/render_queue.c.o
[ 95%] Building C object CMakeFiles/looking-glass-client.dir/src/overlay/splash.c.o
[ 95%] Building C object CMakeFiles/looking-glass-client.dir/src/overlay_utils.c.o
[ 96%] Building C object CMakeFiles/looking-glass-client.dir/src/overlay/alert.c.o
[ 96%] Building C object CMakeFiles/looking-glass-client.dir/src/overlay/fps.c.o
[ 97%] Building C object CMakeFiles/looking-glass-client.dir/src/overlay/config.c.o
[ 97%] Building C object CMakeFiles/looking-glass-client.dir/src/overlay/msg.c.o
[ 97%] Building C object CMakeFiles/looking-glass-client.dir/src/overlay/help.c.o
[ 98%] Building C object CMakeFiles/looking-glass-client.dir/src/overlay/graphs.c.o
[ 99%] Building C object CMakeFiles/looking-glass-client.dir/src/overlay/status.c.o
[100%] Building C object CMakeFiles/looking-glass-client.dir/version.c.o
In file included from /build/source/repos/nanosvg/src/nanosvgrast.h:28,
                 from /build/source/client/src/overlay_utils.c:32:
/build/source/repos/nanosvg/src/nanosvg.h: In function 'nsvg__parseTransform':
/build/source/repos/nanosvg/src/nanosvg.h:1656:15: error: 't[0]' may be used uninitialized [-Werror=maybe-uninitialized]
 1656 |         float t[6];
      |               ^
compilation terminated due to -Wfatal-errors.
cc1: all warnings being treated as errors
make[2]: *** [CMakeFiles/looking-glass-client.dir/build.make:265: CMakeFiles/looking-glass-client.dir/src/overlay_utils.c.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:456: CMakeFiles/looking-glass-client.dir/all] Error 2
make: *** [Makefile:136: all] Error 2

```

</details>

## Additional context

<!-- Add any other context about the problem here. -->

## Metadata

<!-- Please insert the output of running `nix-shell -p nix-info --run "nix-info -m"` below this line -->
 - system: `"x86_64-linux"`
 - host os: `Linux 6.12.5, NixOS, 25.05 (Warbler), 25.05.20241219.d70bd19`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.24.11`
 - nixpkgs: `/etc/nix/path/nixpkgs`

## Notify maintainers

<!--
Please @ people who are in the `meta.maintainers` list of the offending package or module.
If in doubt, check `git blame` for whoever last touched something.
-->

@alexbakker @babbaj @j-brn

---

Note for maintainers: Please tag this issue in your PR.

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/by-name/lo/looking-glass-client/nanosvg-unvendor.diff
+++ b/pkgs/by-name/lo/looking-glass-client/nanosvg-unvendor.diff
@@ -1,0 +1,28 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 3755adc..55e1eb3 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -25,6 +25,7 @@ include(OptimizeForNative) # option(OPTIMIZE_FOR_NATIVE)
+ include(UninstallTarget)
+
+ find_package(PkgConfig)
++find_package(NanoSVG REQUIRED)
+ pkg_check_modules(FONTCONFIG REQUIRED IMPORTED_TARGET fontconfig)
+
+ option(ENABLE_OPENGL "Enable the OpenGL renderer"       ON)
+@@ -106,7 +107,6 @@ include_directories(
+   ${PROJECT_SOURCE_DIR}/include
+   ${CMAKE_BINARY_DIR}
+   ${CMAKE_BINARY_DIR}/include
+-  ${PROJECT_TOP}/repos/nanosvg/src
+ )
+
+ link_libraries(
+@@ -161,6 +161,7 @@ target_compile_definitions(looking-glass-client PRIVATE CIMGUI_DEFINE_ENUMS_AND_
+ target_link_libraries(looking-glass-client
+   ${EXE_FLAGS}
+   PkgConfig::FONTCONFIG
++  NanoSVG::nanosvg
+   lg_resources
+   lg_common
+   displayservers

--- a/pkgs/by-name/lo/looking-glass-client/package.nix
+++ b/pkgs/by-name/lo/looking-glass-client/package.nix
@@ -14,6 +14,7 @@
   libffi,
   expat,
   libGL,
+  nanosvg,
 
   libX11,
   libxkbcommon,
@@ -69,6 +70,7 @@ stdenv.mkDerivation (finalAttrs: {
       hash = "sha256-CqB8AmOZ4YxnEsQkyu/ZEaun6ywpSh4B7PM+MFJF0qU=";
       stripLen = 1;
     })
+    ./nanosvg-unvendor.diff
   ];
 
   nativeBuildInputs = [
@@ -88,6 +90,7 @@ stdenv.mkDerivation (finalAttrs: {
       nettle
       fontconfig
       libffi
+      nanosvg
     ]
     ++ lib.optionals xorgSupport [
       libxkbcommon

